src/save_index.cpp

```diff
@@ -46,6 +46,16 @@ void replace_space2underbar(std::string &name) {
 	std::replace(name.begin(),name.end(),' ','_');
 }
 
+/** Replace the spaces in the filter */
+std::string replace_filter(const std::string* filter)
+{
+    std::string filter_value = *filter;
```

Comment:
I built correctly (with no warnings, moreover) and obviously tested the code. You can be sure of this.